docs/backends/TES.md

```diff
@@ -46,21 +46,23 @@ Currently this backend only works with files on a Local or Shared File System.
 
 **Docker**
 
-This backend supports the following optional [Runtime Attributes](RuntimeAttributes) and [Workflow Options](wf_options/Overview/) for working with Docker:
+This backend supports the following optional [Runtime Attributes](../RuntimeAttributes) and [Workflow Options](../wf_options/Overview/) for working with Docker:
 
 * `docker`: Docker image to use such as "Ubuntu".
 * `dockerWorkingDir`: defines the working directory in the container.
 
 **CPU, Memory and Disk** 
 
-This backend supports CPU, memory and disk size configuration through the use of the following [Runtime Attributes](RuntimeAttributes) and [Workflow Options](wf_options/Overview/):  
+This backend supports CPU, memory and disk size configuration through the use of the following [Runtime Attributes](../RuntimeAttributes) and [Workflow Options](../wf_options/Overview/):  
 
 * `cpu` defines the amount of CPU to use. 
     * Type: Integer (ex: 4)
 * `memory` defines the amount of memory to use. 
     * Type: String (ex: "4 GB" or "4096 MB")
 * `disk` defines the amount of disk to use. 
     * Type: String (ex: "1 GB" or "1024 MB")
+* `preemptible` defines whether or not to use preemptible VMs. 
```

Comment:
I think we'd want preemptible to be a number -- so the number of times one should use a preemptible before reverting to a full size machine in case of failure. This methodology has worked really well for the Google backend.

I would modify the definition to be "defines how many times to use a preemptible VMs" and set default value in the config to 0

Reply:
are there any existing workflows you're looking at that show preemptible being used as true/false?